apple/RNCWebView.m

```diff
@@ -333,6 +333,10 @@ - (WKWebViewConfiguration *)setUpWkWebViewConfig
     [prefs setValue:@TRUE forKey:@"javaScriptCanOpenWindowsAutomatically"];
     _prefsUsed = YES;
   }
+  if (!_textInteractionEnabled) {
+    [prefs setValue:@FALSE forKey:@"textInteractionEnabled"];
+    _prefsUsed = YES;
+  }
```

Comment:
I think this also needs the `#if` check around it?

And possibly also a runtime `if (@available` check?

Reply:
Fixed this with both a #if and @availabel check for iOS 14.5